### ISSUE TITLE
Sanity Checking DataLoader

### ISSUE THREAD

**Issue by hyao1** [lewandofskee/DiAD#46]
![image](https://github.com/lewandofskee/DiAD/assets/52654892/65a26dc8-fac5-4565-8138-b82322e0cf16)
![image](https://github.com/lewandofskee/DiAD/assets/52654892/39159573-f2b6-4b05-a896-3df997073170)

Why are only two images validated during training? There are only 000.jpg and 001.jpg of class bottler are validated. is it right?

**Comment by lewandofskee** [comment]
This is normal. Initially, the Sanity Checking only tests with two images. Subsequent tests are conducted on the entire test set.
